packages/zowe-explorer/src/dataset/DatasetTree.ts

```diff
@@ -159,7 +159,7 @@ export class DatasetTree extends ZoweTreeProvider implements IZoweTree<IZoweData
             }
             const validationStatus = await Profiles.getInstance().checkCurrentProfile(element.getProfile());
 
-            if (validationStatus.status === "unverified") {
+            if (SettingsConfig.getDirectValue(globals.SETTINGS_AUTOMATIC_PROFILE_VALIDATION) && validationStatus.status === "unverified") {
```

Comment:
unverified should still initiate search and either return something or error.  Turning off validation of profile just eliminates the call to check if server is running.

Reply:
The `validationStatus.status === "unverified"` check was added in 40b10157, but I agree, I don't think it's needed. I didn't want to re-introduce the "Update Creds" loop and this was done as part of that PR, so I wanted to make the changes as minimal as possible. I will remove it

Reply:
resolved in [f85c661](https://github.com/zowe/vscode-extension-for-zowe/pull/2461/commits/f85c66102a748b6dc962e84ec38eed2eee59e45c)